### ISSUE TITLE
NixCon 2026 venue proposal: Brussels, Belgium

### ISSUE THREAD

**Issue by drupol** [nixcon/nixcon-proposals#3]
# Venue proposal

![](https://static.wixstatic.com/media/6a4560_58d8e58a7e344e3fa848eacde70106d9~mv2.jpg)

Square – Brussels Meeting Centre has 27 meeting rooms of 40sqm to 1200sqm for a total capacity of 7000 attendees, and an exposition zone of 4000sqm.

Official website: https://square-brussels.com/
Wikipedia: https://en.wikipedia.org/wiki/Square_%E2%80%93_Brussels_Meeting_Centre
Maps: [OpenstreetMap](https://www.openstreetmap.org/way/144502504) /  [Google Maps](https://maps.app.goo.gl/ygsaLN9mDfpap5WG8)

## Context

Having attended events here multiple times, I can attest to its suitability for conferences. Its impressive facilities aside, there are several other benefits:

1. **Location**: In the heart of the capital of Europe, ideal for exploring the city by foot ([map](https://www.google.com/maps/dir/Grand-Place,+Grote+Markt,+Bruxelles/SQUARE+Brussels+Meeting+Centre,+Mont+des+Arts,+Bruxelles/@50.8453896,4.3524943,17z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x47c3c47f4614f1f1:0xb03c355d8fe2cfb6!2m2!1d4.3524356!2d50.8467887!1m5!1m1!1s0x47c3c4873c654411:0x68314d65accae29b!2m2!1d4.3575192!2d50.8438936!3e2?entry=ttu))
2. **Public transport accessibility**:
  - 5-minute walk from the nearest station
  - One train stop away from Brussels Midi international station ([map](https://www.google.com/maps/dir/Bruxelles-Central,+Carr+de+l'Europe,+1000+Bruxelles/SQUARE+Brussels+Meeting+Centre,+Mont+des+Arts,+Bruxelles/@50.8447972,4.3544861,17z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x47c3c47fcc1d13b5:0x40b1a1704a249cfa!2m2!1d4.3570964!2d50.845504!1m5!1m1!1s0x47c3c4873c654411:0x68314d65accae29b!2m2!1d4.3575192!2d50.8438936!3e2?entry=ttu))
  - 25 minutes by train from the airport ([link](https://irail.be/route?to=http%3A%2F%2Firail.be%2Fstations%2FNMBS%2F008819406&from=http%3A%2F%2Firail.be%2Fstations%2FNMBS%2F008813003&date=041223&time=1422&timeSel=depart))
3. **Accommodations**: numerous hotels nearby, including Hilton, Novotel, Agora, Ibis, NH, Thon, etc.
4. **Parking**: available within a 5-minute walk ([map](https://maps.app.goo.gl/pixRUWj5nNdZ7BNW7))
5. **Local curiosities**:
  - The Grand Place ([link](https://en.wikipedia.org/wiki/Grand-Place)), the Hotel de Ville ([map](https://maps.app.goo.gl/47JAcEffvj71XMMh9))
  - The King's Palace ([map](https://maps.app.goo.gl/XMqWrhXVMWSmbyRp6))
  - The Royal Park ([map](https://maps.app.goo.gl/Dq2j9dTh6dmjd9EA6))
  - The Cathedral Saint Michel ([map](https://maps.app.goo.gl/mRS37Pmry9SuUcFW6))
  - Magritte museum ([map](https://maps.app.goo.gl/XYfVRUsM1n2u1mSD8))
  - Natural sciences museum ([map](https://maps.app.goo.gl/Q1GzkQH3AbYdT13L9))
  - Musical instruments museum ([map](https://maps.app.goo.gl/hv327eJMoZEgtKAK7))
  - Manneken Pis ([map](https://maps.app.goo.gl/U5BvAMcVQeMefsTF9))
  - Europeen Commission area and headquarters ([map](https://maps.app.goo.gl/N8hwdQkKDkvkbLtM7)) 
  - Cinquantenaire ([map](https://maps.app.goo.gl/UZoBfduW4jCxFaKKA))
  - European Parliament ([map](https://maps.app.goo.gl/qJXERHHxAZkpTWwX8))
  - The Atomium easily accessible with the subway ([map](https://maps.app.goo.gl/HCNBLhyvcY8kfusy9))

## Statistics

This section has been achieved with the help of Eva, an employee of the venue.

- Total capacity: 7000
- Capacity in the main track room: 1200
- A/V equipment in the track room, e.g. Projector: Yes
- Hacking day capacity: Yes
- Number of tracks possible: 10 (at the same time) (_need clarification on this_), 
- Hallway track possible? Yes
- Mini-conference (BoFs) tracks possible? (< 100 persons => nope)
- Distance (time) from an airport or train station in public transports: 25 min by train - ([link](https://irail.be/route?to=http%3A%2F%2Firail.be%2Fstations%2FNMBS%2F008819406&from=http%3A%2F%2Firail.be%2Fstations%2FNMBS%2F008813003&date=041223&time=1422&timeSel=depart))
- Average price for a hotel room in the city: < 200€
- Potential costs to pay to the venue owner: /

## Pictures

- From the architect: https://www.a2rc.be/copie-de-hotel-de-ligne
- From Google Images: https://www.google.com/search?client=firefox-b-d&sca_esv=587665533&q=brussels+Square+conference+center&tbm=isch&source=lnms&sa=X&ved=2ahUKEwin37Lj7fWCAxVxdqQEHcIAA2sQ0pQJegQIDhAB&biw=1600&bih=829&dpr=1.2
- A couple of pictures I took on the 7-8 December 2023, during SymfonyCon:

<img src="https://github.com/nixcon/nixcon-proposals/assets/252042/91d637f0-bb96-4485-b4f2-baeb429a66fd" alt="drawing" width="200"/>
<img src="https://github.com/nixcon/nixcon-proposals/assets/252042/a58b9939-74d2-44ea-ba79-a35bfdf6af52" alt="drawing" width="200"/>
<img src="https://github.com/nixcon/nixcon-proposals/assets/252042/2c398bd1-493d-4260-be1b-785bbcdad034" alt="drawing" width="200"/>
<img src="https://github.com/nixcon/nixcon-proposals/assets/252042/0b677c15-0696-4316-97c7-0148b4baec23" alt="drawing" width="200"/>

**Comment by asymmetric** [comment]
@drupol is this still WIP, or have you gotten all the info you wanted to get?

**Comment by drupol** [comment]
Yep I have all the infos I need. I don't have time for working on this now, but I will continue in ~3 weeks.

**Comment by AkechiShiro** [comment]
@drupol any news ?

**Comment by drupol** [comment]
Nothing concrete on my side any more. To go further with this venue, an official quote needs to be sent, which is out of my responsibility at this point.

**Comment by infinisil** [comment]
I guess we can close this now unless you want to propose this for 2026 as well :)

**Comment by drupol** [comment]
Yes, it is still valid for 2026 ! If there's anything to do to push this forward, just let me know.

**Comment by infinisil** [comment]
Ahh then I'll update the title :)

**Comment by tomberek** [comment]
Selected Poland for this iteration, but thank you for the proposal!
